docs/integration-guides/websockets.md

```diff
@@ -19,9 +19,7 @@ Sample clients are available:
 
 ## Configuration
 
-For details on configuring websockets within a node, see the [websocket section of Running a Node Configuration](/running-a-node/configuration#websocket).
-
-With the default configuration, localhost clients should connect to `ws://[::1]:7078`.
```

Comment:
Provided the websocket is enabled (either by default with your recent PR or manually), is `ws://[::1]:7078` not a correct default endpoint?

Reply:
As the default configuration is different on Docker vs non Docker, "with the default configuration" could be ambiguous if you can connect to this url out of the box. Furthermore, the URL to connect to is also stated in /running-a-node/configuration#websocket so no need to maintain URL consistency between the two pages.
I mentioned this in my commit message but I should have mentioned that again in the PR.

Reply:
I should have read the commit messages, different flow than I'm used to. Thanks for clearing it up!